wnp-server/src/main/java/com/wonnapark/wnpserver/domain/episode/application/EpisodeFind.java

```diff
@@ -0,0 +1,18 @@
+package com.wonnapark.wnpserver.domain.episode.application;
+
+import com.wonnapark.wnpserver.domain.episode.dto.response.EpisodeDetailFormResponse;
+import com.wonnapark.wnpserver.domain.episode.dto.response.EpisodeListFormResponse;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+public interface EpisodeFind {
```

Comment:
[nit] UseCase라는 PostFix는 어떤가요
```suggestion
public interface EpisodeFindUseCase {
```

Reply:
UseCase를 붙여서 실제 행위가 아닌 사용 사례임을 명시적으로 나타내주어서 좋네요